on_the_porch/unused/data_download/import_911_to_mysql.py

```
@@ -0,0 +1,316 @@
+import os
+from datetime import datetime
+import glob
+import numpy as np
+
+def connect_to_mysql():
+    """Connect to MySQL database"""
+    try:
+        connection = mysql.connector.connect(
+            host='localhost',
+            user='rethinkai_user',
+            password='MySecureUserPass123!',
```

Ruff (0.14.1)

Possible hardcoded password assigned to argument: "password"

(S106)